### ISSUE TITLE
Set up ETD work type

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/mahonia#9]
**ACCEPTANCE CRITERIA**
- [x] There is an ETD work type
- [x] It supports the basic Hyrax Metadata   
- [x] I can attach a PDF
- [x] The PDF is set as the representative file 
- [x] The PDF is set at the thumbnail
~~- [ ] I can attach additional files (like tabular data)~~
- [x] Public access ETDs are visible to non-authenticated users
- [x] Administrators can edit ETD metadata after creation
- [x] Non-restricted PDFs can be downloaded

**Comment by no-reply** [comment]
This is in progress at https://github.com/curationexperts/mahonia/tree/etd-work-type

**Comment by no-reply** [comment]
I believe we need a `AssignPdfAsRepresentativeActor`/`AssignPdfAsThumbnailActor` to satisfy those requirements when an additional file is uploaded. Hyrax default behavior makes the first `FileSet` processed the representative/thumbnail.

**Comment by no-reply** [comment]
I have questions about a couple of the remaining requirements here:

> Public access ETDs are visible to non-authenticated users

Is the requirement that non-public ETDs should not be visible? I'm inclined test both?

----

>  Administrators can edit ETD metadata after creation

What is an administrator? Is this blocked pending #20, or should this be part of that ticket (or a new one)?

I'm inclined to test `Edit` for a work I just submitted and call this complete until we've had discussions about the abilities of the admin role.

----

>  Non-restricted PDFs can be downloaded

What is a non-restricted PDF? Just one with `visibility` set to `:public`, or... ?

**Comment by no-reply** [comment]
@mark-dce I'd like to close this ticket on merge of #80. 

Several questions arose for me in the process of implementing and testing this (see https://github.com/curationexperts/mahonia/issues/9#issuecomment-338300245). The status after #80 is:

  - There is an ETD work type:
    - **In place and tested**.
  - It supports the basic Hyrax Metadata:
    - **In place; tests are somewhat thin** but are better than anything I've seen us do in the past and can be extended if needed during the project.
  -  I can attach a PDF:
     - **In place and tested**. Feature tests are in place, and a factory exists for doing this programmatically in the test suite.
  - The PDF is set as the representative file:
    -  **In place and tested**; however, this doesn't hold if supplemental files are added. See https://github.com/curationexperts/mahonia/issues/9#issuecomment-338300245. I would like to **open a new ticket** for this.
  - I can attach additional files (like tabular data)
    - In place but **not tested**. I would slate testing this work for the ticket proposed in the above bullet.
  - Public access ETDs are visible to non-authenticated users:
    - **In place and tested** for both search and item pages. We do not currently test that private items are not visible. **I would like to discuss item visibility requirements before putting more work in here**.
  - Administrators can edit ETD metadata after creation:
    - There are currently **no administrators pending #20**. That depositors/owners can edit items is **in place, tested, and factory-ized**.
  - Non-restricted PDFs can be downloaded.
    - **In place and tested**. We do not currently test that private items are not downloadable. **I would like to discuss item visibility requirements before putting more work in here**.

**Comment by no-reply** [comment]
Tickets are open for the items that came up during implementation of this. I believe #80 genuinely closes this ticket at this point.
